cognite/client/data_classes/data_modeling/data_types.py

```diff
@@ -48,78 +46,98 @@ def as_tuple(self) -> tuple[str, str]:
 @dataclass
 class PropertyType(ABC):
     _type: ClassVar[str]
+    is_list: bool = False
 
     def dump(self, camel_case: bool = True) -> dict[str, Any]:
         output = asdict(self)
         output["type"] = self._type
         for key in list(output.keys()):
             if output[key] is None:
                 output.pop(key)
-        output = rename_and_exclude_keys(output, aliases=_PROPERTY_ALIAS_INV)
+        output = rename_and_exclude_keys(output, aliases=_PROPERTY_ALIAS)
         return convert_all_keys_recursive(output, camel_case)
 
+    @staticmethod
+    def __load_unit_ref(data: dict) -> UnitReference | None:
+        unit: UnitReference | None = None
+        if (unit_raw := data.get("unit")) and isinstance(unit_raw, dict):
+            unit = UnitReference.load(unit_raw)
+        return unit
+
     @classmethod
     def load(cls, data: dict) -> Self:
-        if "type" not in data:
-            raise ValueError("Property types are required to have a type")
         type_ = data["type"]
```

Comment:
To handle the case when there is not list passed.
```suggestion
        type_ = data["type"]
        is_list = data.get("list", False)
```